.semaphore/semaphore.yml

```diff
@@ -168,7 +168,7 @@ blocks:
 
         - name: KStreams Serialization tests
           commands:
-            - sem-version java 11
+            - sem-version java 17
```

Comment:
I think we might be able to get rid of this one since it will use the same version of Java for all the tests.